awscli/topics/return-codes.rst

```diff
@@ -31,8 +31,23 @@ of a CLI command:
 
 * ``130`` -- The process received a SIGINT (Ctrl-C).
 
-* ``255`` -- Command failed. There were errors thrown by either the CLI or
-  by the service the request was made to.
+* ``252`` -- Command syntax was invalid, an unknown parameter was provided, or
+  a paramter value was incorrect and prevented the command from running.
+
+* ``253`` -- The system environment or configuration was invalid. While the
+  command provided may be syntactically valid, missing configuration or
+  credentials prevented the command from running.
+
+* ``254`` -- The command was succesfully parsed and a request was made to the
+  specified service but the service returned an error. This will generally
+  indicate incorrect API usage or other service specific issues and is likely
```

Comment:
The "is likely not an issue with the request being sent by the CLI" is sort of weird phrasing. Honestly, we can probably just remove that last part.